views/login.handlebars

```diff
@@ -56,8 +56,8 @@
                         <label for="password" class="cols-sm-2 control-label">Password</label>
                         <div class="cols-sm-10">
                             <div class="input-group">
-                                <span class="input-group-addon"><i class="fa fa-lock fa-lg" aria-hidden="true"></i></span>
-                                <input type="password" class="form-control" name="password" id="password" placeholder="Enter your password" />
+                                <span class="input-group-addon" data-toggle="tooltip" title="A strong password contains at least 8 characters, 1 number, 1 capital letter and 1 symbol"><i class="fa fa-lock fa-lg" aria-hidden="true"></i></span>
```

Comment:
I'm gonna include the Very Weak condition in it, but other than that I believe it to be fine